manim/mobject/logo.py

```diff
@@ -107,6 +109,7 @@ def scale(self, scale_factor: float, **kwargs) -> "ManimBanner":
             self.anim.scale(scale_factor, **kwargs)
         return super().scale(scale_factor, **kwargs)
 
+    @override_animation(Create)
     def create(self, run_time: float = 2) -> AnimationGroup:
```

Comment:
I'll note here that I'm not sure whether this method should be private or not. For backwards compatibility this is probably fine as is, and for harmony with the `expand` method, but I wonder if maybe ultimately users shouldn't call this method directly.

Reply:
I think it should be `create` for the same reasons.